examples/test/test_qnn_tooklit.py

```diff
@@ -11,35 +13,54 @@
 from olive.common.utils import retry_func, run_subprocess
 
 
-def download_qnn_sdk():
+def download_qnn_sdk(target_path=None):
     """Download the qnn sdk."""
+    blob, download_path = "", ""
+    if platform.system() == "Windows":
+        blob, download_path = "qnn_sdk_windows.zip", "qnn_sdk_windows.zip"
+    elif platform.system() == "Linux":
+        blob, download_path = "qnn_sdk_linux.zip", "qnn_sdk_linux.zip"
+
     download_azure_blob(
         container="olivetest",
-        blob="qnn_sdk_linux.zip",
-        download_path="qnn_sdk_linux.zip",
+        blob=blob,
+        download_path=download_path,
     )
-    target_path = Path().resolve()
-    run_subprocess(cmd=f"unzip qnn_sdk_linux.zip -d {str(target_path)}", check=True)
+    if not target_path:
+        target_path = Path().resolve() / "qnn_sdk"
+    target_path.mkdir(parents=True, exist_ok=True)
+    if platform.system() == "Windows":
+        cmd = f"powershell Expand-Archive -Path {download_path} -DestinationPath {str(target_path)}"
+        run_subprocess(cmd=cmd, check=True)
+    elif platform.system() == "Linux":
+        run_subprocess(cmd=f"unzip {download_path} -d {str(target_path)}", check=True)
     return target_path
 
 
 @pytest.fixture(scope="module", autouse=True)
 def setup():
     """Setups any state specific to the execution of the given module."""
+    tmp_dir = tempfile.TemporaryDirectory()
+    tmp_path = Path(tmp_dir.name)
```

Comment:
use tmp_path fixture instead?

Reply:
https://stackoverflow.com/questions/62335029/how-do-i-use-tmpdir-with-my-pytest-fixture 
Seems can not use it here as the tmp_dir is passed into another function of `download_qnn_sdk`.

That leads the error: https://aiinfra.visualstudio.com/PublicPackages/_build/results?buildId=401411&view=logs&j=4f5f4a75-935b-5877-7f37-0e73f78960b0&t=b01a3c21-1243-5721-75b4-0ba5e3889b9d